aws-ecr-replicationconfiguration/src/main/java/software/amazon/ecr/replicationconfiguration/ListHandler.java

```diff
@@ -9,20 +11,51 @@
 import software.amazon.cloudformation.proxy.ProxyClient;
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
+import java.util.Collections;
+import java.util.List;
+
 public class ListHandler extends BaseHandlerStd {
```

Comment:
Did you want the replication configuration code in this commit as well?

Reply:
removed